recOrder/calib/Calibration.py

```diff
@@ -639,13 +645,41 @@ def opt_I135(self, lca_bound, lcb_bound):
         logging.info("--------done--------")
         logging.debug("--------done--------")
 
-    def calc_blacklevel(self):
+    def open_shutter(self):
+        if self.shutter_device == '': # no shutter
+            input('Please manually open the shutter and press <Enter>')
+        else:
+            self.mmc.setShutterOpen(True)
+
+    def reset_shutter(self):
+        """
+        Return autoshutter to its original state before closing
 
-        auto_shutter = self.mmc.getAutoShutter()
-        shutter = self.mmc.getShutterOpen()
+        Returns
+        -------
 
-        self.mmc.setAutoShutter(False)
-        self.mmc.setShutterOpen(False)
+        """
+        if self.shutter_device == '': # no shutter
+            input('Please reset the shutter to its original state and press <Enter>')
```

Comment:
Do we want to direct the user back to the GUI here? (e.g. `logging.info("This is the end of the commandline instructions.")`)

Reply:
Great idea!